src/app/content/hooks/receiveContent.ts

```diff
@@ -66,4 +66,8 @@ const hookBody: ActionHookBody<typeof receivePage | typeof locationChange> = (se
   dispatch(setHead({links, meta, title}));
 };
 
+const hookBody: ActionHookBody<typeof receivePage | typeof locationChange> = (services) => {
+  return async() => updateHead(services);
+};
+
```

Comment:
I'm not certain if there is a better way to accomplish this, I needed to extract the body due to the types expecting the action to be passed, but I need to call it directly in locationChange without having the page info

Reply:
where you're calling this here https://github.com/openstax/rex-web/pull/1779/files#diff-5c521d5b56bf1fc8410db4d851fabef38392a7ee950c8d6d314d7a317d8d5336R25 you have the locationChange action passed in on line 17

Reply:
Thank you I totally missed that :(